NativeApp/src/components/Login/Form/index.js

```diff
@@ -1,8 +1,7 @@
 import React, { Component } from 'react';
-import { Button } from 'react-native-elements';
+import { Button, FormValidationMessage, FormInput, FormLabel } from 'react-native-elements';
```

Comment:
Es linter will complain about the length of this line. Do something like

```javascript
import {
   Button,
   FormValidationMessage,
   FormInput,
   FormLabel 
} from 'react-native-elements';
```
Plus make it look tidy. :)

Reply:
i have made requested changes :)